pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+      return;
+    }
+
+    if (paymentDetails.paymentMethod === null) {
+      toast({ status: 'error', description: 'Missing payment method', isClosable: true });
+      return;
+    }
+
+    await submitNewApplication({
+      variables: {
+        input: {
+          firstName: permitHolderInformation.firstName,
+          middleName: permitHolderInformation.middleName,
+          lastName: permitHolderInformation.lastName,
+          email: permitHolderInformation.email,
+          phone: permitHolderInformation.phone,
+          receiveEmailUpdates: permitHolderInformation.receiveEmailUpdates,
+          addressLine1: permitHolderInformation.addressLine1,
+          addressLine2: permitHolderInformation.addressLine2,
+          city: permitHolderInformation.city,
+          postalCode: permitHolderInformation.postalCode,
+          dateOfBirth: permitHolderInformation.dateOfBirth,
+          gender: permitHolderInformation.gender,
+          otherGender: permitHolderInformation.otherGender,
+
+          disability: physicianAssessment.disability,
+          disabilityCertificationDate: physicianAssessment.physicianCertificationDate,
+          patientCondition: physicianAssessment.patientCondition,
+          mobilityAids: null, //TODO: get mobility aids when forms are updated to get this data
+          otherPatientCondition: physicianAssessment.patientEligibilityDescription || null,
+          permitType: physicianAssessment.permitType,
+          temporaryPermitExpiry: physicianAssessment.temporaryPermitExpiryDate,
+
+          physicianFirstName: doctorInformation.firstName,
+          physicianLastName: doctorInformation.lastName,
+          physicianMspNumber: doctorInformation.mspNumber,
+          physicianPhone: doctorInformation.phone,
+          physicianAddressLine1: doctorInformation.addressLine1,
+          physicianAddressLine2: doctorInformation.addressLine2,
+          physicianCity: doctorInformation.city,
+          physicianPostalCode: doctorInformation.postalCode,
+
+          omitGuardianPoa: guardianInformation.omitGuardianPoa,
+          guardianFirstName: guardianInformation.firstName,
+          guardianMiddleName: guardianInformation.middleName,
+          guardianLastName: guardianInformation.lastName,
+          guardianPhone: guardianInformation.phone,
+          guardianRelationship: guardianInformation.relationship,
+          guardianAddressLine1: guardianInformation.addressLine1,
+          guardianAddressLine2: guardianInformation.addressLine2,
+          guardianCity: guardianInformation.city,
+          guardianPostalCode: guardianInformation.postalCode,
+          poaFormUrl: guardianInformation.poaFormUrl,
+
+          ...additionalQuestions,
+          usesAccessibleConvertedVan: additionalQuestions.usesAccessibleConvertedVan,
+          requiresWiderParkingSpace: additionalQuestions.requiresWiderParkingSpace,
```

Comment:
This should be covered by `...additionalQuestions`

Reply:
Because these fields are initialized to null, we have to explicitly specify them. They are checked for `null` values before calling `submitNewApplication`, but the null check isn't recognized without including these here.

```
pages/admin/request/create-new.tsx:421:9 - error TS2322: Type '{ paymentMethod: PaymentType; paidThroughShopify: false; shopifyPaymentStatus: null; shopifyConfirmationNumber: null; applicantId: number | null; donationAmount: string; ... 59 more ...; otherGender: Maybe<...>; }' is not assignable to type 'CreateNewApplicationInput'.
  Types of property 'usesAccessibleConvertedVan' are incompatible.
    Type 'boolean | null' is not assignable to type 'boolean'.
      Type 'null' is not assignable to type 'boolean'.

421         input: {
```